misc/tests/conftest.py

```diff
@@ -0,0 +1,8 @@
+def pytest_addoption(parser):
+    parser.addoption("--suppress", action="append", default=[],
+                     help="Suppression files used in test-memory-leaks.py. These files must be located in the folder valgrind.")
```

Comment:
Wouldn't it be simpler to just pass full paths to suppression files here rather than requiring that they are in a specific directory?